sycl/test-e2e/FreeFunctionKernels/SeparateCompilation.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %{build} %S/SumKernel.cc %S/ProductKernel.cc -o %t.out
```

Comment:
How does it work if `%s` isn't used on this command line, i.e. when the binary being built doesn't have `main` function? Am I missing something?